.gitignore

```diff
@@ -2,6 +2,7 @@
 # will have compiled files and executables
 /target/
 /sqlparser_bench/target/
+/derive/target/
```

Comment:
Also note to self

I tested using

```shell
cargo test --all --features=visitor
```

We need to add this feature to the CI tests

Reply:
Oh, that explains the poor coverage I guess

Reply:
I verified that the CI does run with `--all-features`: https://github.com/sqlparser-rs/sqlparser-rs/actions/runs/3794845435/jobs/6453351547

https://github.com/sqlparser-rs/sqlparser-rs/blob/dec3c2b818bdedcc81f0e3990e2dd2fefcf9e872/.github/workflows/rust.yml#L67